test/inductor/test_max_autotune.py

```diff
@@ -1290,6 +1290,43 @@ def foo(x, y):
             "tl.store"
         ).run(code[0])
 
+    @config.patch(
+        {
+            "max_autotune_gemm_backends": "Triton",
+            "benchmark_epilogue_fusion": True,
+            "use_mixed_mm": False,
+            "mixed_mm_choice": "default",
+        }
+    )
+    def test_pending_fusions(self):
+        def multi_use(x, y):
+            return (x @ x.T) * (y @ y.T)
+
+        x = torch.rand([128, 16], device="cuda")
```

Comment:
Since this test suit is shared by GPUs(CUDA, XPU), suggest to use `GPU_TYPE` instead of `cuda` here,  or decorate this case as requires_cuda.

Otherwise this case will failed on XPU. Thanks.